src/target/riscv/riscv.c

```diff
@@ -619,8 +619,17 @@ static int find_first_trigger_by_id(struct target *target, int unique_id)
 	return -1;
 }
 
+static unsigned int count_trailing_ones(riscv_reg_t reg)
+{
+	for (int i = 0; i < 64; i++) {
```

Comment:
Small detail: It would be good to make the data type match the return type of the function.

```suggestion
	for (unsigned int i = 0; i < 64; i++) {
```

Reply:
addressed